src/main.jsx

```diff
@@ -1,10 +1,11 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
-import './index.css'
-import App from './App.jsx'
+import 'bootstrap/dist/css/bootstrap.css'
+import 'bootstrap/dist//js/bootstrap.bundle.js'
```

Comment:
it will not work as 'bootstrap/dist//js/bootstrap.bundle.js' there is double // in path.
also if you are not using any bootstrap js functions, then remove it. Becasue I am not able to see any code related to it. @kaushaki-kumari